Snowflake.Data/Snowflake.Data.csproj

```diff
@@ -1,106 +1,38 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{3E6328DD-E284-4F23-B55A-472BB7EA2C25}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Snowflake.Data</RootNamespace>
-    <AssemblyName>Snowflake.Data</AssemblyName>
-    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
-    <TargetFrameworkProfile />
+    <TargetFrameworks>netstandard2.0;net46</TargetFrameworks>
+		<Title>Snowflake.Data</Title>
+    <Description>Snowflake Connector for .NET</Description>
+    <Company>Snowflake Computing, Inc</Company>
+    <Product>Snowflake Connector for .NET</Product>
+    <Copyright>Copyright (c) 2012-2018 Snowflake Computing Inc. All rights reserved.</Copyright>
+		<Version>0.3.0</Version>

```

Comment:
If this is the only version info, this is fine. But if multiple places have the version info, it would be better to have a separate version file so that all can share the same.

Reply:
This should be the only place for version. 

The other version should not be needed since that is the project file for tests binaries.

Reply:
ok, by production, add a log including the driver name and version so that the given log file tell what version of driver is used.